website/src/controllers/api-v3/quests.js

```diff
@@ -65,4 +67,54 @@ api.inviteToQuest = {
   },
 };
 
+/**
+ * @api {post} /groups/:groupId/quests/abort Abort a quest
+ * @apiVersion 3.0.0
+ * @apiName AbortQuest
+ * @apiGroup Group
+ *
+ * @apiParam {string} groupId The group _id (or 'party')
+ *
+ * @apiSuccess {Object} Quest Object
+ */
+api.abortQuest = {
+  method: 'POST',
+  url: '/groups/:groupId/quests/abort',
+  middlewares: [authWithHeaders(), cron],
+  async handler (req, res) {
+    // Abort a quest AFTER it has begun (see questCancel for BEFORE)
+    let user = res.locals.user;
+    let groupId = req.params.groupId;
+
+    req.checkParams('groupId', res.t('groupIdRequired')).notEmpty();
+
+    let validationErrors = req.validationErrors();
+    if (validationErrors) throw validationErrors;
+
+    let group = await Group.getGroup({user, groupId, fields: 'type quest'});
+    if (!group) throw new NotFound(res.t('groupNotFound'));
+    if (!group.quest.active) throw new NotFound(res.t('noActiveQuestToAbort'));
+
+    let memberUpdates = User.update(
+      {'party._id': groupId},
+      {
+        $set: {'party.quest': Group.cleanQuestProgress()},
+        $inc: {_v: 1},
```

Comment:
@TheHollidayInn I'm adding a TODO to my list so we can standardize the _v usage since it's not increased everywhere and it probably fits in an update middleware http://mongoosejs.com/docs/middleware.html